README.md

```diff
@@ -4,44 +4,41 @@
 
 RESTEasy is a JBoss.org project aimed at providing productivity frameworks for developing client and server RESTful applications and services in Java.  It is mainly a JAX-RS implementation but you'll find some other experimental code in the repository.
 
-The project page can be found at https://resteasy.jboss.org
+The project page can be found at [https://resteasy.github.io/](https://resteasy.github.io/).
```

Comment:
I believe we can leave the resteasy.jboss.org link there (we have a redirect in place).

Reply:
yes sir